pennylane/ops/qutrit/observables.py

```diff
@@ -133,3 +149,199 @@ def compute_diagonalizing_gates(eigenvectors, wires):  # pylint: disable=argumen
+        Implicitly, this assumes that the wires of the operator correspond to the global wire order.
+
+        .. seealso:: :meth:`~.GellMann.matrix`
+
+        Args:
+            index (int): The index of the Gell-Mann matrix to be used. Must be between 1
+            and 8 inclusive
+
+        Returns:
+            tensor_like: canonical matrix
+
+        **Example**
+
+        >>> qml.GellMann.compute_matrix(8)
+        array([[ 0.57735027+0.j,  0.        +0.j,  0.        +0.j],
+               [ 0.        +0.j,  0.57735027+0.j,  0.        +0.j],
+               [ 0.        +0.j,  0.        +0.j, -1.15470054+0.j]])
+        """
+        return gm_mats[index - 1]
+
+    @staticmethod
+    def compute_eigvals(index):  # pylint: disable=arguments-differ
+        r"""Eigenvalues of the operator in the computational basis (static method).
+
+        If :attr:`diagonalizing_gates` are specified and implement a unitary :math:`U`,
+        the operator can be reconstructed as
+
+        .. math:: O = U \Sigma U^{\dagger},
+
+        where :math:`\Sigma` is the diagonal matrix containing the eigenvalues.
+
+        Otherwise, no particular order for the eigenvalues is guaranteed.
+
+        .. seealso:: :meth:`~.GellMann.eigvals`
+
+        Args:
+            index (int): The index of the Gell-Mann matrix to be used. Must be between 1
+            and 8 inclusive
+
+        Returns:
+            array: eigenvalues
+
+        **Example**
+
+        >>> qml.GellMann.compute_eigvals(1)
+        [1. -1.  0.]
+        """
+        if index != 8:
+            return np.array([1, -1, 0])
+
+        return np.array([1, 1, -2]) / np.sqrt(3)
+
+    @staticmethod
+    def compute_diagonalizing_gates(
+        wires, index
+    ):  # pylint: disable=arguments-differ,unused-argument
+        r"""Sequence of gates that diagonalize the operator in the computational basis (static method).
+
+        Given the eigendecomposition :math:`O = U \Sigma U^{\dagger}` where
+        :math:`\Sigma` is a diagonal matrix containing the eigenvalues,
+        the sequence of diagonalizing gates implements the unitary :math:`U`.
+
+        The diagonalizing gates rotate the state into the eigenbasis
+        of the operator.
+
+        .. seealso:: :meth:`~.GellMann.diagonalizing_gates`.
+
+        Args:
+            index (int): The index of the Gell-Mann matrix to be used. Must be between 1 and 8 inclusive
+            wires (Iterable[Any], Wires): wires that the operator acts on
+        Returns:
+            list[.Operator]: list of diagonalizing gates
+
+        **Example**
+
+        >>> qml.GellMann.compute_diagonalizing_gates(wires=0, index=4)
+        [QutritUnitary(array([[ 0.70710678-0.j,  0.        -0.j,  0.70710678-0.j],
+               [ 0.70710678-0.j,  0.        -0.j, -0.70710678-0.j],
+               [ 0.        -0.j,  1.        -0.j,  0.        -0.j]]), wires=[0])]
+        """
+        if index in {3, 8}:
+            return []
```

Comment:
Adding a comment for devs, 
```suggestion
            return []  # already diagonal
```